langserve/api_handler.py

```diff
@@ -1070,15 +1070,23 @@ async def create_feedback(
             comment=feedback_from_langsmith.comment,
         )
 
-    async def check_feedback_enabled(self, config_hash: str = "") -> None:
-        """Check if feedback is enabled for the runnable."""
-        if not tracing_is_enabled() or not self._enable_feedback_endpoint:
+    async def _check_feedback_enabled(self, config_hash: str = "") -> None:
+        """Check if feedback is enabled for the runnable.
+
+        This endpoint is private since it will be deprecated in the future.
+
+        """
+        if not self._enable_feedback_endpoint or not tracing_is_enabled():
```

Comment:
swap order of conditional for short circuiting -- self._enable_feedback_endpoint is a local attribute, with `tracing_is_enabled()` could include some hidden cost (at the moment there's no difference)